.github/workflows/chart-test.yaml

```diff
@@ -27,6 +27,9 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.6.1
 
+      - name: Symlink ci-lint -> ci
+        run: ln -s ./ci-lint charts/rstudio-workbench/ci
```

Comment:
what is the working directory?

Reply:
The working directory is the root of the repo.  I could instead do `ln -s ~/charts/rstudio-workbench/ci-lint charts/rstudio-workbench/ci` if that's your concern. 

The relative link seemed cleaner to me, but I am happy to change it if you like

```walk.go:74: found symbolic link in path: /home/runner/work/helm/helm/charts/rstudio-workbench/ci resolves to /home/runner/work/helm/helm/charts/rstudio-workbench/ci-install
``` is how it looks from `ct`'s point of view

Reply:
Ah, okay, I think?